avm/res/compute/virtual-machine/version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-  "version": "0.10",
+  "version": "0.11",
```

Comment:
Please update to 0.12. In the meantime, a new version of the VM was published.

Reply:
Done.